lib/main-menu.js

```diff
@@ -60,6 +60,13 @@ var file = {
       }
     },
     {
+      label: 'Focus Note',
+      accelerator: 'Control + E',
+      click: function () {
```

Comment:
I follow the code style in the same file. Should I change all of them to ES2015+ style?